RedHat-UX/red-hat-design-system
# fix: add breakpoint images from Tokens repo

## What I did

1. Created a new folder called `breakpoints` under `docs/assets`
2. Copy breakpoint SVGs from the Tokens repo into that folder
3. Fix relative path to breakpoint images


## Testing Instructions

1. Check that images are showing on [breakpoints token](https://deploy-preview-1344--red-hat-design-system.netlify.app/tokens/breakpoint) page in the DP

## Notes to Reviewers

This relies on [#120 ](https://github.com/RedHat-UX/red-hat-design-tokens/pull/120) getting merged first.
